azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -50,6 +52,13 @@ public void run() throws Exception {
       handleError("Bad property definition! " + e.getMessage(), e);
     }
 
+    Pair<Long, Long> memPair = getProcMemoryRequirement();
```

Comment:
It will be a good I idea to have a config to enable disable this feature from azkaban.properties or even curl requests